tax_reform_explorer/data.py

```diff
@@ -126,15 +126,17 @@ def main(name, reform):
     print_data(calcbase, calcreform, weights = weighted, tab = 'c00100', name=name)
 
 if __name__ == '__main__':
-    reform_values = (0, .5, 1)
+    #reform_values = (0, .5, 1)
+    reform_values = (1,)
     groups = {}
-    for i in range(3):
-        for j in range(3):
-            for k in range(3):
+    for i in range(1):
+        for j in range(1):
+            for k in range(1):
                 reform = {
                           2015:{
                         "_ID_InterestPaid_HC":[reform_values[i]],
                         "_ID_StateLocalTax_HC":[reform_values[j]],
+                        "_ID_RealEstate_HC":[reform_values[j]],
```

Comment:
This is the right parameter. Then the single toggle for state and local taxes in the widget should control both `_ID_StateLocalTax_HC` as well as `_ID_RealEstateHC`.